src/Microsoft.SqlTools.ServiceLayer/TableDesigner/Contracts/Requests/GeneratePreviewReportRequest.cs

```diff
@@ -22,6 +23,11 @@ public class GeneratePreviewReportResult
         /// format (mimetype) of the string
         /// </summary>
         public string MimeType;
+
+        /// <summary>
+        /// Metadata about the table to be captured
```

Comment:
what does `to be captured` mean?  please also update the comment in ads, i think you have the same comment there.